src/metrics.cc

```diff
@@ -190,6 +190,13 @@ Metrics::Metrics()
                     "microseconds.")
               .Register(*registry_)),
 
+      // Versions
+      version_info_family_(
+          prometheus::BuildGauge()
+            .Name("triton_version_information")
```

Comment:
Hi @ClifHouck, thanks for the contrbution.

I reached out on the original issue to clarify the ask - I'm not sure if server/backend API versions are necessary if the actual triton release version is reflected and can be mapped.

However, it does raise a question that this info may be useful to add to an existing endpoint. Do you have any reasoning on exposing these through prometheus rather than another existing REST endpoint? Pros of adding it to a standard (non-metrics) endpoint is that it would always be available, regardless of whether the server was built or configured at runtime to use prometheus metrics or not.